### DISCUSSION TITLE
问一下大佬们，现在v2ray 5.0.3不是支持fullcone吗，为什么netch测试为unsuporttedserver

### DISCUSSION THREAD

**Discussion by GleenJi** [v2fly/v2ray-core#1699] [Q&A] [answered]
如题

**Comment by AkinoKaede** [comment, marked as answer]
You shoud set `packetEncoding` setting of inbounds to `Packet`.

For example:

```json
{
    "inbounds": [
        {
            "protocol": "socks",
            "settings": {
                "udpEnabled": true,
                "packetEncoding": "Packet"
            },
            "port": 1080
        }
    ],
    "outbounds": [
        {
            "protocol": "vmess",
            "settings": {
                "address": "v2fly.org",
                "port": 0,
                "uuid": "00000000-0000-0000-0000-000000000000"
            }
        }
    ]
}
```

**Reply by GleenJi** [reply]
> You shoud set `packetEncoding` setting of inbounds to `Packet`.
> 
> For example:
> 
> ```json
> {
>     "inbounds": [
>         {
>             "protocol": "socks",
>             "settings": {
>                 "udpEnabled": true,
>                 "packetEncoding": "Packet"
>             },
>             "port": 1080
>         }
>     ],
>     "outbounds": [
>         {
>             "protocol": "vmess",
>             "settings": {
>                 "address": "v2fly.org",
>                 "port": 0,
>                 "uuid": "00000000-0000-0000-0000-000000000000"
>             }
>         }
>     ]
> }
> ```

v2ray v5的websocket的设置里，原本有个headers: {Host: ""}，在v4中没问题，v5中换成了什么格式呀？

**Reply by AkinoKaede** [reply]
#1599

**Reply by GleenJi** [reply]
> #1599

非常非常感谢大佬！！

**Reply by GleenJi** [reply]
> #1599

不过还是有个问题，就是服务端一套上warp，udp用netch测试为中等，并且netch测试出为cf的ip（outbounds里的sendThrough设定为vps的ip了），一关闭warp为fullcone，，，，，，，用xray的话使用（outbounds里的sendThrough设定为vps的ip）netch测试udp就为fullcone

**Comment by Remonli** [comment]
5.0.3 现在是什么状态呀？可以用了吗？没看到文档更新，新版本配置文件怎么写。

**Reply by GleenJi** [reply]
> 5.0.3 现在是什么状态呀？可以用了吗？没看到文档更新，新版本配置文件怎么写。

可以用，但是由于目前v5文档不全并且存在一些错误，还是先使用v4吧

**Comment by 1715173329** [comment]
dokodemo 入站依旧无法 fullcone，能支持下么？
